packages/core/contracts/optimistic-asserter/implementation/escalation-manager/FullPolicyEscalationManager.sol

```diff
@@ -0,0 +1,181 @@
+pragma solidity 0.8.16;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "./BaseEscalationManager.sol";
+import "../../interfaces/OptimisticAsserterInterface.sol";
+
+enum AssertionBlockMode {
+    None, // No assertions are blocked
+    BlockByAssertingCallerAndAsserter, // Assertion are block by asserting caller and asserter pair
+    BlockByAssertingCaller // Assertions are blocked by asserting caller
+}
+
+/**
+ * @title The FullPolicyEscalationManager allows the owner to configure all Escalation Manager policy parameters and store
+ * arbitration resolutions. Optionally, we can enable assertion blocking via a whitelist of assertingCallers or a
+ * combination of a whitelist of assertingCallers and asserters. On the other hand, it allows us to determine if we want
```

Comment:
Consider not using the word "us" but rather "owner". Same applies throughout the comments.